### DISCUSSION TITLE
How can I change Material Bottom Tab active indicator color?

### DISCUSSION THREAD

**Discussion by arambrosius** [react-navigation/react-navigation#11299] [Q&A] [answered]
I have always used react navigation's material bottom tab navigator for all my react-native projects but I am getting a different behavior on a new one compared to any of the others. I can't find it in this repo anymore though so I'm not sure where to find the answer. I followed the [docs](https://reactnavigation.org/docs/material-bottom-tab-navigator/) but I am still getting a purple circle that I can't change the color to. 

How can I change this Active Indicator color?

It looks like this was introduced in [material 3 ](https://m3.material.io/components/navigation-bar/specs) but I don't see any props on this package that I can change to update this value.

<img width="426" alt="Screenshot 2023-03-22 at 8 38 22 PM" src="https://user-images.githubusercontent.com/35042830/227080680-3e4e9e73-2ad9-4cd7-9796-08ad7ba7dc59.png">

**Comment by BeeMargarida** [comment]
Since this uses react-native-paper BottomNavigator, there's a way of changing it that was already asked over there.

Following the provided answer in [react-native-paper/#3566](https://github.com/callstack/react-native-paper/issues/3566), you need to change the secondary theme color like `theme={{colors: {secondaryContainer:  'your color' }}}`.

**Comment by arambrosius** [comment, marked as answer]
Thank you. I was still using React Navigations material bottom tab navigator rather than using React Native Paper's. Once I switched to that it worked perfectly. Thanks for the quick help.
